RecoEgamma/EgammaElectronProducers/python/gsfElectrons_cfi.py

```diff
@@ -115,14 +117,9 @@
     recHitSeverityToBeExcludedEndcaps = cleanedHybridSuperClusters.RecHitSeverityToBeExcluded,
     #severityLevelCut = cms.int32(4),
 
-    # Isolation algos configuration
-    intRadiusBarrelTk = cms.double(0.015), 
-    intRadiusEndcapTk = cms.double(0.015), 
-    stripBarrelTk = cms.double(0.015), 
-    stripEndcapTk = cms.double(0.015), 
-    ptMinTk = cms.double(0.7), 
-    maxVtxDistTk = cms.double(0.2), 
-    maxDrbTk = cms.double(999999999.), 
+    # Isolation algos configuration 
+    trkIsol03Cfg = trkIsol03CfgV2,
+    trkIsol04Cfg = trkIsol04CfgV2,                                        
```

Comment:
what is the reason to have different isolation definition here, compared to the gedGsfElectrons?

Also, shouldn't the PSet for gsfElectrons below be updated as well?

Reply:
if V1 is supposed to produce the same result as the baseline isolation, perhaps this can be changed to V1 
in order to not disturb the inputs to PF reco.
I'd rather we do not make any changes to PF algorithms now due to possible validation implications.

Reply:
This was a mistake, thank you for your careful reading and catch here. I've set it to V3. GsfElectrons arent really used for much these days. 

So the PF algos take gedGsfElectrons (specifically gedGsfElectronsTmp). Yes changing the track isolation will potentially have a change on which electrons are identified as PF electrons or not. Let me put your mind at ease that this is not a problem

How this works:
PFEgammaFilters decides if an electron is isolated or not by looking at the sum of tracker , ecal and hcal isolations. If this value is less than 10 GeV (no even rho corrected), it uses the isolated electron MVA to decide if its an electron. Otherwise it uses the unisolated electron MVA

https://github.com/cms-sw/cmssw/blob/master/RecoParticleFlow/PFProducer/src/PFEGammaFilters.cc#L99-L115
 
So to me, changing one part of the sum slightly (and likely due to pileup, the smallest part of the sum) which is used to decide which MVA should be used (for which the criteria already arbitrary and not well motivated at all) will have no significant impact. I just mention it as its the only other place in CMSSW that I'm aware of it being used. If it does have any significant impact, we have bigger problems as it means a variable which is not PU corrected with a cut which is not well motivated is playing a major role in our physics output.